crates/node/ops/mod.rs

```diff
@@ -7,8 +7,12 @@ pub mod http2;
 pub mod idna;
 pub mod ipc;
 pub mod os;
+pub mod process;
 pub mod require;
 pub mod util;
 pub mod v8;
+pub mod vm;
```

Comment:
do we need to expose `vm`? Probably need to stay disabled.

Reply:
Sure